datastore/uncached.go

```diff
@@ -6,57 +6,70 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+// GetLonghornEventList returns a list of longhorn events for the given namespace
 // Event list is uncached but current lister doesn't field selector, so we have no choice but read from API directly
-
 func (s *DataStore) GetLonghornEventList() (*corev1.EventList, error) {
 	return s.kubeClient.CoreV1().Events(s.namespace).List(metav1.ListOptions{FieldSelector: "involvedObject.apiVersion=longhorn.io/v1beta1"})
 }
 
 // The following APIs are not cached, should only be used in e.g. the support bundle
 
+// GetAllPodsList returns a list of pods for the given namespace
```

Comment:
```suggestion
// GetAllPodsList returns an uncached list of pods for the given namespace directly from the api server.
// Using cached informers should be prefered and direct retrieval from the api server should only be used for one shot tasks for example support bundle creation.
```

@c3y1huang Thank you for the PR's it's appreciated 👍 
Would you mind adding more details (Why, Why, When, etc) for example to the comments, like in the above suggestion?
Currently your comments only describe the `What` part which is self explanatory based on the Function name.

Reply:
Thank you for the review and suggestion. I've addressed in https://github.com/longhorn/longhorn-manager/pull/655/commits/05580128f7e6ad437ded4cd3ddddcc40d59e8b49, PTAL.